CHANGELOG/v1.9.0.md

```diff
@@ -0,0 +1,448 @@
+- KCP: Drop unnecessary etcd call from KCP (#11493)
+- KCP: Ensure ReadinessGates for v1beta2 conditions get set (#11335)
+- KCP: Propagate timeouts to Machines with deletionTimestamp (#11128)
+- KCP: Refine KCP's Available, ControlPlaneComponentsHealthy, EtcdClusterHealthy v1beta2 conditions (#11512)
+- KCP: Refine v1beta2 Available condition in KCP (#11425)
+- KCP: Refine v1beta2 ControlPlaneHealthy condition (#11438)
+- KCP: Refine v1beta2 KCP available condition (#11451)
+- KCP: Remove code handling Kubernetes <= v1.21 (#11146)
+- KCP: Retry in case of etcd errors in KCP (#11450)
+- KCP: Skip validation if CoreDNS migration library supports an upgrade if the library is not used (#11319)
+- Logging: Drop internal log package & improve logs and errors (#11025)
+- Logging: Drop ReconcileError events (#11341)
+- Logging: Fix error handling when the resource is not found (#10907)
+- Logging: Improve log when not enough replicas are ready or referenced (#11329)
+- Logging: Log controller name in predicates (#10959)
+- Logging: Log error when calling a Runtime Extension gets an error that is ignored because of failure policy (#11028)
+- Logging: Reduce verbosity of logs when calling Runtime Extensions (#11183)
+- Machine: Add feature gate to consider VolumeAttachments when waiting for volume detach (#11386)
+- Machine: Condition: fix godoc for MachineNodeHealthyCondition (#11178)
+- Machine: Extend Node drain e2e test to cover MachineDrainRules (#11362)
+- Machine: Follow-up PR nits from 11032 (#11044)
+- Machine: Implement Machine Deleting condition (#11291)
+- Machine: Improve Drain for control plane machines (#11457)
+- Machine: Improve machine Ready v1beta2 condition (#11330)
+- Machine: Improve node drain e2e test (#11127)
+- Machine: Improve unit tests for Machine controller (#11252)
+- Machine: Prevent error spamming for NodeOutdatedTaint if objects are not found (#11148)
+- Machine: Rate-limit entire reconcileDelete (#11436)
+- Machine: Refine v1beta2 machine ready (#11437)
+- Machine: Refine v1beta2 NodeHealthy condition (#11497)
+- Machine: Use internal error reason consistently (#11309)
+- MachineDeployment: Refine MachineDeployment v1beta2 available condition (#11505)
+- MachineHealthCheck: Increase timeout for 2 MHC unit tests (#11019)
+- MachinePool: Improve TestReconcileMachinePoolMachines unit test (#11124)
+- MachineSet: Add MD watch to the MS controller (#11448)
+- MachineSet: Extend MS ScalingUp and Remediationg conditions to include preflight check errors (#11390)
+- MachineSet: Implement MS remediating conditions (#11382)
+- Metrics: Add nilIsZero to all KSM metric configs where needed (#11101)
+- Metrics: Revert "Add nilIsZero to all KSM metric configs where needed" (#11107)
+- ClusterClass/MachinePool: Remove paused handling from reconcileExternal (#11400)
+- KCP/MachineSet: Implement KCP and MS Deleting conditions (#11381)
+- Machine/KCP]: Implement grace period for KCP remote conditions (#11339)
```

Comment:
Please remove "]" 
```suggestion
- Machine/KCP: Implement grace period for KCP remote conditions (#11339)
```

Reply:
ohh! might be it got left while updating